### PR TITLE
[StreamAnalytics] upgrade version to 2.0.0

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

+ Generated code using swagger json.
+ Customed cmdlet.
+ Add examples and test cases.

### CODE REVIEW

File: src/StreamAnalytics/Az.StreamAnalytics.psd1 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@{
+  GUID = 'f3678192-db41-439b-99e7-6fda95f6c601'
+  RootModule = './Az.StreamAnalytics.psm1'
+  ModuleVersion = '2.0.1'
```
Comment (dingmeng-xue) [inline src/StreamAnalytics/Az.StreamAnalytics.psd1:4]: should be 2.0.0

File: src/StreamAnalytics/Az.StreamAnalytics.psd1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  FormatsToProcess = './Az.StreamAnalytics.format.ps1xml'
+  FunctionsToExport = 'Get-AzStreamAnalyticsCluster', 'Get-AzStreamAnalyticsClusterStreamingJob', 'Get-AzStreamAnalyticsDefaultFunctionDefinition', 'Get-AzStreamAnalyticsFunction', 'Get-AzStreamAnalyticsInput', 'Get-AzStreamAnalyticsJob', 'Get-AzStreamAnalyticsOutput', 'Get-AzStreamAnalyticsQuota', 'Get-AzStreamAnalyticsTransformation', 'New-AzStreamAnalyticsCluster', 'New-AzStreamAnalyticsFunction', 'New-AzStreamAnalyticsInput', 'New-AzStreamAnalyticsJob', 'New-AzStreamAnalyticsOutput', 'New-AzStreamAnalyticsTransformation', 'Remove-AzStreamAnalyticsCluster', 'Remove-AzStreamAnalyticsFunction', 'Remove-AzStreamAnalyticsInput', 'Remove-AzStreamAnalyticsJob', 'Remove-AzStreamAnalyticsOutput', 'Start-AzStreamAnalyticsJob', 'Stop-AzStreamAnalyticsJob', 'Test-AzStreamAnalyticsFunction', 'Test-AzStreamAnalyticsInput', 'Test-AzStreamAnalyticsOutput', 'Update-AzStreamAnalyticsCluster', 'Update-AzStreamAnalyticsFunction', 'Update-AzStreamAnalyticsInput', 'Update-AzStreamAnalyticsJob', 'Update-AzStreamAnalyticsOutput', 'Update-AzStreamAnalyticsTransformation', '*'
+  AliasesToExport = '*'
+  PrivateData = @{
+    PSData = @{
```
Comment (dingmeng-xue) [inline src/StreamAnalytics/Az.StreamAnalytics.psd1:17]: Add one tag for preview release Prerelease = 'preview'

reference https://www.powershellgallery.com/packages/Az.Compute/4.0.1-preview/Content/Az.Compute.psd1

File: src/StreamAnalytics/Az.StreamAnalytics.psd1 (hunk 1 above)
Comment (LucasYao93) [reply src/StreamAnalytics/Az.StreamAnalytics.psd1]: Done

File: src/StreamAnalytics/Az.StreamAnalytics.psd1 (hunk 2 above)
Comment (LucasYao93) [reply src/StreamAnalytics/Az.StreamAnalytics.psd1:17]: Xiaogang will create new branch from the generation branch to resolve it.

**Review by dolauli — APPROVED** [review]
LGTM

**Pull request merged**